blade-graphics/src/gles/command.rs

```diff
@@ -806,6 +810,65 @@ impl super::Command {
                     None,
                 );
             }
+
+            Self::BlitFramebuffer { from, to } => {
+                /*
+                    TODO(ErikWDev): Validate
+                */
+
+                let target_from = match from.inner {
+                    super::TextureInner::Renderbuffer { raw } => raw,
+                    _ => panic!("Unsupported resolve between non-renderbuffers"),
```

Comment:
Is this just a thing we want to leave for a follow-up, or you are seeing actual technical difficulty here? I think, handling a texture here should be straightforward.

Reply:
Nop, just unfamiliar with opengl. Found framebufferTexture2D so should be easy

Reply:
Think this should work
```rust
                match from.inner {
                    super::TextureInner::Renderbuffer { raw } => {
                        gl.framebuffer_renderbuffer(
                            glow::READ_FRAMEBUFFER,
                            glow::COLOR_ATTACHMENT0, // NOTE: Assuming color attachment
                            glow::RENDERBUFFER,
                            Some(raw),
                        );
                    }
                    super::TextureInner::Texture { raw, target } => {
                        gl.framebuffer_texture_2d(
                            glow::READ_FRAMEBUFFER,
                            glow::COLOR_ATTACHMENT0,
                            glow::RENDERBUFFER,
                            Some(raw),
                            0,
                        );
                    }
                }
```

Reply:
yep! if in doubt, we can also check wgpu-hal